warehouse/templates/manage/account/publishing.html

```diff
@@ -134,33 +134,18 @@ <h3>GitHub</h3>
         {{ field_errors(pending_github_publisher_form.project_name) }}
       </div>
     </div>
-    <div class="form-group">
-      <label for="owner" class="form-group__label">
-        {% trans %}Owner{% endtrans %}
-        {% if pending_github_publisher_form.owner.flags.required %}
-        <span class="form-group__required">{% trans %}(required){% endtrans %}</span>
-        {% endif %}
-      </label>
-      {{ pending_github_publisher_form.owner(placeholder=gettext("owner"), autocomplete="off", autocapitalize="off", spellcheck="false", class_="form-group__field", aria_describedby="owner-errors") }}
-      <p class="form-group__help-text">
-        {% trans %}The GitHub organization name or GitHub username that owns the repository{% endtrans %}
-      </p>
-      <div id="owner-errors">
-        {{ field_errors(pending_github_publisher_form.owner) }}
-      </div>
-    </div>
     <div class="form-group">
       <label for="repository" class="form-group__label">
-        {% trans %}Repository name{% endtrans %}
+        {% trans %}Repository{% endtrans %}
         {% if pending_github_publisher_form.repository.flags.required %}
         <span class="form-group__required">{% trans %}(required){% endtrans %}</span>
         {% endif %}
       </label>
-      {{ pending_github_publisher_form.repository(placeholder=gettext("repository"), autocomplete="off", autocapitalize="off", spellcheck="false", class_="form-group__field", **{"aria-describedby":"repository-errors"}) }}
+      {{ pending_github_publisher_form.owner(placeholder=gettext("repository"), autocomplete="off", autocapitalize="off", spellcheck="false", class_="form-group__field", aria_describedby="owner-errors") }}
```

Comment:
Let's update the placeholder text to hint at the format we're expecting.

```suggestion
      {{ pending_github_publisher_form.owner(placeholder=gettext("user/repo"), autocomplete="off", autocapitalize="off", spellcheck="false", class_="form-group__field", aria_describedby="owner-errors") }}
```